aiida/cmdline/params/options/__init__.py

```diff
@@ -8,590 +8,111 @@
-                help_string = f'Whether to expand the node set by following {TRAVERSAL_RULE_HELP_STRING[name]}.'
-                click.option(option_label, default=traversal_rule.default, show_default=True, help=help_string)(command)
-
-        return command
-
-    return decorator
-
-
-PROFILE = OverridableOption(
-    '-p',
-    '--profile',
-    'profile',
-    type=types.ProfileParamType(),
-    default=defaults.get_default_profile,
-    help='Execute the command for this profile instead of the default profile.'
-)
-
-CALCULATION = OverridableOption(
-    '-C',
-    '--calculation',
-    'calculation',
-    type=types.CalculationParamType(),
-    help='A single calculation identified by its ID or UUID.'
-)
-
-CALCULATIONS = OverridableOption(
-    '-C',
-    '--calculations',
-    'calculations',
-    type=types.CalculationParamType(),
-    cls=MultipleValueOption,
-    help='One or multiple calculations identified by their ID or UUID.'
-)
-
-CODE = OverridableOption(
-    '-X', '--code', 'code', type=types.CodeParamType(), help='A single code identified by its ID, UUID or label.'
-)
-
-CODES = OverridableOption(
-    '-X',
-    '--codes',
-    'codes',
-    type=types.CodeParamType(),
-    cls=MultipleValueOption,
-    help='One or multiple codes identified by their ID, UUID or label.'
-)
-
-COMPUTER = OverridableOption(
-    '-Y',
-    '--computer',
-    'computer',
-    type=types.ComputerParamType(),
-    help='A single computer identified by its ID, UUID or label.'
-)
-
-COMPUTERS = OverridableOption(
-    '-Y',
-    '--computers',
-    'computers',
-    type=types.ComputerParamType(),
-    cls=MultipleValueOption,
-    help='One or multiple computers identified by their ID, UUID or label.'
-)
-
-DATUM = OverridableOption(
-    '-D', '--datum', 'datum', type=types.DataParamType(), help='A single datum identified by its ID, UUID or label.'
-)
-
-DATA = OverridableOption(
-    '-D',
-    '--data',
-    'data',
-    type=types.DataParamType(),
-    cls=MultipleValueOption,
-    help='One or multiple data identified by their ID, UUID or label.'
-)
-
-GROUP = OverridableOption(
-    '-G', '--group', 'group', type=types.GroupParamType(), help='A single group identified by its ID, UUID or label.'
-)
-
-GROUPS = OverridableOption(
-    '-G',
-    '--groups',
-    'groups',
-    type=types.GroupParamType(),
-    cls=MultipleValueOption,
-    help='One or multiple groups identified by their ID, UUID or label.'
-)
-
-NODE = OverridableOption(
-    '-N', '--node', 'node', type=types.NodeParamType(), help='A single node identified by its ID or UUID.'
-)
-
-NODES = OverridableOption(
-    '-N',
-    '--nodes',
-    'nodes',
-    type=types.NodeParamType(),
-    cls=MultipleValueOption,
-    help='One or multiple nodes identified by their ID or UUID.'
-)
-
-FORCE = OverridableOption('-f', '--force', is_flag=True, default=False, help='Do not ask for confirmation.')
+# yapf: disable
```

Comment:
Since we are removing all other code from the init's and just have the `__all__`, why do we need to disable and reenable `yapf`? I know that `yapf` would change the one resource per line to just try and fit as much on one line, but why is this not desirable?

Reply:
Well firstly IMO, it makes it less readable, but mainly I think it would make the pre-commit hooks clash, since each would keep changing them back and forth

Reply:
In the spirit of this PR, I'm happy with the explicit nature of using `# yapf: disable`, rather than relying on some obscure interpretation of commas by yapf. So we'll agree to disagree on this one 😉